### ISSUE TITLE
Training datasets

### ISSUE THREAD

**Issue by mtaktash** [lck666666/plana3r#7]
Hi! Thanks for your work

What is the exact combination of the training datasets you use from Habitat that you mention in the paper. As I see (screenshot below), it supports both Habitat-Matterport3D, Matterport3D, Gibson and Replica, but in the paper you explore generalization on Matterport3D and Replica. 

Could you also share the split of the scenes you have trained on for ScanNet dataset? Is it all the scenes not included in SparsePlanes ScanNet test set?

Thanks!

<img width="790" height="217" alt="Image" src="https://github.com/user-attachments/assets/ca5712dc-30be-4ba9-b05e-fbffd8f843ee" />
